PubSub/PubSub.csproj

```diff
@@ -10,13 +10,13 @@
     </UpgradeBackupLocation>
     <OldToolsVersion>12.0</OldToolsVersion>
     <!--Start of Nuspec metadata-->
-    <PackageId>PubSub</PackageId>
-    <Title>PubSub dotnet</Title>
-    <Authors>upta</Authors>
-    <PackageProjectUrl>http://github.com/upta/pubsub</PackageProjectUrl>
+    <PackageId>dt.PubSub</PackageId>
+    <Title>dt.PubSub dotnet</Title>
+    <Authors>tobuto, upta</Authors>
+    <PackageProjectUrl>https://github.com/devterm-its/dt.PubSub</PackageProjectUrl>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <PackageTags>portable;pubsub;eventaggregator;c#</PackageTags>
-    <Description>An extremely light-weight, easy to use .Net pub/sub library</Description>
+    <Description>An extremely light-weight, easy to use .Net pub/sub library based on upta/pubsub</Description>
     <Version>4.0.0</Version>
```

Comment:
NuGet published version is actually 4.0.1, but upta's source on GitHub still shows 4.0.0.

Regardless, if you intend to publish this to NuGet, it should warrant at least a minor, if not a major, version increment, no?

Reply:
Oh, I thought I had changed all version tags in the file, must have missed this one!